src/utils/constants.ts

```diff
@@ -1,7 +1,3 @@
-export const exchangeContractSrc =
-  "usjm4PCxUd5mtaon7zc97-dt-3qf67yPyqgzLnLqk5A";
-export const exchangeWallet = "aLemOhg9OGovn-0o4cOCbueiHT9VgdYnpJpq7NgMA1A";
-
```

Comment:
Could've still been in `constants.ts` and used as default values when argument is not provided.

Reply:
Done